.github/workflows/release_community_cromwell.yml

```diff
@@ -0,0 +1,99 @@
+name: Release Community Cromwell
+
+on:
+  workflow_dispatch: 
+
+jobs:
+  cromwell-release:
+    name: Release Community Cromwell
+    runs-on: ubuntu-latest
+    steps:
+    - uses: sbt/setup-sbt@v1
+
+    # `DSDEJENKINS_PASSWORD` auto syncs from vault with https://github.com/broadinstitute/terraform-ap-deployments/pull/614
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: dsdejenkins
+        password: ${{ secrets.DSDEJENKINS_PASSWORD }}
+
+    - name: Find Cromwell release number
+      run: |
+        set -e
+        previous_version=$(curl -X GET https://api.github.com/repos/broadinstitute/cromwell/releases/latest | jq .tag_name | xargs)
+        if ! [[ "${previous_version}" =~ ^[0-9][0-9]+$ ]]; then
+          exit 1
+        fi
+        echo "RELEASE_VERSION=$((previous_version + 1))" >> $GITHUB_ENV
+        echo "NEXT_VERSION=$((previous_version + 2))" >> $GITHUB_ENV
+
+    - name: Clone Cromwell
+      uses: actions/checkout@v4
+      with:
+        repository: broadinstitute/cromwell
+        token: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
+        path: cromwell
+        ref: develop
+        fetch-depth: 0 # <-- important for branch shenanigans below
```

Comment:
Could you provide a bit more details about this? I could not pin point exactly where this is used below

Reply:
Sure, will expand comment - basically this tells the action to check out all the past commits, which is important for git to understand how to merge branches together.

Reply:
I think in this case `0` is a sentinel for infinity.